site/community/release_guide.md

```diff
@@ -447,6 +450,18 @@ Once all issues have been resolved, you should go back and build a new release c
 
 Once the release candidate has been reviewed and approved by the community, the release should be finalized. This involves the final deployment of the release candidate to the release repositories, merging of the website changes, etc.
 
+### Advance version on release branch
```

Comment:
can't maven:release plugin handle this?

Reply:
AFAIK this is done done, in fact Jia for 4.6 release left the version to "4.6.0-SNAPSHOT" and I left it to "4.5.1-SNAPSHOT" and JV to "4.5.0-SNAPSHOT"
so some command is missing, or maybe there is another better way to bump version

Reply:
I think mvn:release does bump the major version when creating the branch. I need to check out the commands.

Reply:
anyway in my example I am using the mvn release plugin with the "update-versions" mojo

Reply:
and after releasing 4.6.0 (major version) you should have 4.6.1-SNAPSHOT, maybe Jia ans JV used the wrong command

Reply:
Right, it is create when we creating the branch.